google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/MessageDispatcher.java

```diff
@@ -426,21 +431,37 @@ int computeDeadlineSeconds() {
 
   @InternalApi
   void extendDeadlines() {
-    List<String> acksToSend = Collections.<String>emptyList();
-    PendingModifyAckDeadline modack = new PendingModifyAckDeadline(getMessageDeadlineSeconds());
+    int extendSeconds = getMessageDeadlineSeconds();
+    PendingModifyAckDeadline modack = new PendingModifyAckDeadline(extendSeconds);
+    ArrayListMultimap<Long, String> expiringModack = ArrayListMultimap.create();
```

Comment:
Why use this multimap instead of just adding these to the modacks list directly?

Reply:
Done